install_config/certificate_customization.adoc

```diff
@@ -103,6 +103,62 @@ the server to pick up the configuration changes.
 
 For the master API or web console, wildcard names are accepted.
 
+[[configuring-custom-certificates-wildcard]]
+== Configuring a Custom Wildcard Certificate for the Default Router
+
+You can configure the {product-title} default router with a default wildcard certificate. A default wildcard certificate provides a convenient way for 
+applications that are deployed in {product-title} to use default encryption without needing custom certificates. 
+
+[NOTE]
+====
+Default wildcard certificates are recommended for non-production environments only.
+====
+
+In order to configure a default wildcard certificate, provision a certificate that is valid for `.<app_domain>`, where `<app_domain>` is the value of `openshift_master_default_subdomain` in the xref:../install_config/install/advanced_install.adoc#configuring-ansible[Ansible inventory file], by default `/etc/ansible/hosts`. Once provisioned, place the certificate, key, and ca certificate files on your Ansible host, and add the following line to your Ansible inventory file.
+
+----
+openshift_hosted_router_certificate={"certfile": "/path/to/apps.c1-ocp.myorg.com.crt", "keyfile": "/path/to/apps.c1-ocp.myorg.com.key", "cafile": "/path/to/apps.c1-ocp.myorg.com.ca.crt"}
+----
+
+For example:
+
+----
+openshift_hosted_router_certificate={"certfile": "/home/cloud-user/star-apps.148.251.233.173.nip.io.cert.pem", "keyfile": "/home/cloud-user/star-apps.148.251.233.173.nip.io.key.pem", "cafile": "/home/cloud-user/ca-chain.cert.pem"}
+----
+
+Where the parameter values are: 
+
+* *certfile* is the path to the file that contains the {product-title} router certificate.
+
+* *keyfile* is the path to the file that contains the {product-title} wildcard key.
+
+* *cafile* is the path to the file that contains the root CA for this key and certificate. If an intermediate CA is in use, the file should contain both the intermediate and root CA.
+
+If these certificate files are new to your {product-title} cluster, run the Ansible *_deploy_cluster.yml_* playbook to add these files to the {product-title} configuration files. 
```

Comment:
I think if only want to add the customized router cert files for an existing cluster, no need to run "deploy_cluster.yml", "playbooks/openshift-hosted/deploy_router.yml" playbook is enough.

Reply:
BTW, the playbook path is different per version, the above path is only for 3.10.

Reply:
@jianlinliu, does running "deploy_cluster.yml" apply to all versions or just 3.6?

What's the right path for other versions?

Reply:
"deploy_cluster.yml" only apply to 3.9 and 3.10; 3.6 and 3.7 are using "playbooks/byo/config.yml"

Reply:
I'm a little confused. I've changed the reference to `deploy_cluster.yml` to `/byo/config.yml` in 3.6 and 3.7. Please let me know if you want me to change all the versions to use `playbooks/openshift-hosted/deploy_router.yml` instead.

Reply:
If you prefer to use the overall entry playbook, its path should be:
3.9 ~ 3.10: playbooks/deploy_cluster.yml
3.6 ~ 3.7:  playbooks/byo/config.yml
Since 3.9, we introduce standalone plabybook, no need to re-run the overall entry playbook, so for 3.9 ~ 3.10, it is enough to run the standalone playbook - "playbooks/openshift-hosted/deploy_router.yml".
BTW, seen from the PR, I do not know what chnage would applied per different of version, so maybe though you follow my suggest, but I did not get it.

Reply:
Thank you so much!
- I opened a new PR to change 3.9 and 3.10 to use deploy_router.yml here: https://github.com/openshift/openshift-docs/pull/10101
- I fixed this PR for 3.6 and 3.7 to use byo/config.yml.

Is this all correct now?